elements/pfe-navigation/demo/index_react.html

```diff
@@ -11,27 +11,25 @@
   <script type="text/babel">
     function ReactApp() {
       return (
-        <pfe-navigation id="pfe-navigation">
-          <div className="pfe-navigation__overlay"></div>
-
-          <nav className="pfe-navigation" aria-label="Main Navigation">
-            <div className="pfe-navigation__logo-wrapper" id="pfe-navigation__logo-wrapper">
-              <a href="#" className="pfe-navigation__logo-link">
-                <img className="pfe-navigation__logo-image pfe-navigation__logo-image--small" src="assets/redhat--reverse.svg" width="400" alt="Redhat" />
+        <pfe-navigation id="pfe-navigation-small">
+          <nav class="pfe-navigation" aria-label="Main Navigation">
+            <div class="pfe-navigation__logo-wrapper" id="pfe-navigation__logo-wrapper">
+              <a href="#" class="pfe-navigation__logo-link">
+                <img class="pfe-navigation__logo-image pfe-navigation__logo-image--small" src="assets/redhat--reverse.svg" width="400" alt="Red Hat" />
               </a>
             </div>
-            <ul className="pfe-navigation__menu" id="pfe-navigation__menu">
-              <li className="pfe-navigation__menu-item">
-                <a href="#" className="pfe-navigation__menu-link" aria-haspopup="true" aria-expanded="false">
+            <ul class="pfe-navigation__menu" id="pfe-navigation__menu">
+              <li class="pfe-navigation__menu-item">
+                <a href="#" class="pfe-navigation__menu-link">
                   Products
                 </a>
 
-                <div className="pfe-navigation__dropdown">
-                  <section>
+                <div class="pfe-navigation__dropdown pfe-navigation__12-column-grid-wrapper">
```

Comment:
I think that might need to be `className`

Reply:
Probably something all of the `class` attributes need for React in this file.

Reply:
Thanks for catching! I will get that updated.